DEVELOPMENT.md

```diff
@@ -113,9 +113,13 @@ helm uninstall -n kubearchive kubearchive
    ```bash
    kubectl port-forward -n kubearchive svc/kubearchive-database 5432:5432
    ```
-2. Populate the database with test objects.
+1. Run the sql script to create the `resource` table.
    ```bash
-   go run database/init_db.go
+   psql -h localhost -U kubearchive -p 5432 -d kubearchive --password -a -q -f database/init_db.sql
```

Comment:
About the name of the file `init_db.sql`.
If we keep `dml-example.sql` I would suggest to keep `ddl.sql` for naming.

I prefer DDL, that stands for "Data Definition Language" as it's more accurate than "Initialize Database".
The reason is that "Initialize Database" could literally mean to run the [CREATE DATABASAE command](https://www.postgresql.org/docs/current/sql-createdatabase.html) that we are actually not running (we are actually expecting to have the database already created, the one that is called `kubearchive` to which we are connecting in this line.

If we change the name to `ddl.sql` this other comment won't longer be applicable but I suggest to stick with "hyphens" for naming files instead of underscores.
We are using `dml-example.sql` so I think we should have `init-db.sql` instead of `init_db.sql`

Reply:
I second the request to use "hyphens" in the name. I also agree that the file should not be named "init-db", since there can be other files that will be run to initialize aspects of the database. My suggested name will be added as a comment on the file itself.

Reply:
Hey! I renamed the sql script. Following your suggestions @maruiz93 @beerparty Thanks!